blogposts/2020/can-you-hear-me-no-but-we-read-you.md

```diff
@@ -0,0 +1,96 @@
+
+Thankfully, I have a wonderful, supportive team, and this has never been a problem in my day-to-day work.
+
+Our team has been growing quickly, expanding from just a single product manager to a team of eight product managers and designers ([we’re still growing!](https://about.sourcegraph.com/company/careers)), and we’re always thinking of ways to connect and support each other as a team. So when our VP of Product suggested an idea to help the team build empathy and better understand the video call experience from the perspective of someone relying on captions, we jumped on it.
+
+When we held our next product team sync, the whole team joined sans audio. We would all depend only on captions to follow the conversation.
+
+<blockquote style="border-left: 2px solid #F96216;">
+<p><strong>A quick caution</strong></p>
+<p>Most exercises to simulate the experience of living with a disability aren’t effective—or even a good idea—because they can never truly simulate what it’s like to live day-by-day, and can leave folks with a mistaken idea of the experience.</p>
+<p>But this idea was compelling because the unique nature of the hacky setup I’m using means that it’s not about <em>ability</em>, it’s about <em>technology</em>. It wasn’t about learning what it’s like to do video calls with a hearing problem: it was about learning what it’s like to rely on captions for video calls.</p>
+</blockquote>
+
+## What we did
+
+The team was committed. Our meeting document was stuffed with meaningful topics that we wanted to cover, and we expected to use the full hour set aside for our sync.
+
+We ended up using two different video calling platforms and two different approaches to captioning.
+
+The first attempt combined [Zoom](http://zoom.us/) and [Otter](https://otter.ai). Each person opened a browser window with the transcript, and located it somewhere on their screen so they could see the call along with the “captions.” This transcript didn’t include speaker names, so folks had to pay close attention to who was talking and map it to the transcript.
+
+This worked well until it didn’t. Within five minutes, the transcript lagged a good 5-10 seconds behind the conversation. In my own day-to-day setup, I often use multiple speech-to-text tools simultaneously for this exact reason, so I suggested we jump over to [Google Meet](https://meet.google.com), which has captions built right in.
+
+The team immediately spotted the tradeoffs: while Meet’s captions were built in, there’s no history, meaning each person had to dedicate their attention to the captions to make sure they didn’t miss anything. And there’s a few UI issues that create friction or interrupt the captions.
+
+But fundamentally, it _worked_. Everyone was able to participate in the call, and we had our sync.
+
+<img src="/blog/silent-team-call.png" alt="The team on the video call" />
+
+## How it went
```

Comment:
But weird to read this headline after reading "But fundamentally, it _worked_." in the previous paragraph. 

I'd either get rid of the header or rename it to "Takeways" or something like that.